flytekit/configuration/__init__.py

```diff
@@ -218,7 +218,11 @@ def validate_image(_: typing.Any, param: str, values: tuple) -> ImageConfig:
         return ImageConfig.create_from(default_image=default_image, other_images=images)
 
     @classmethod
-    def create_from(cls, default_image: Image, other_images: typing.Optional[typing.List[Image]] = None) -> ImageConfig:
+    def create_from(
+        cls, default_image: Optional[Image], other_images: typing.Optional[typing.List[Image]] = None
+    ) -> ImageConfig:
+        if default_image and not isinstance(default_image, Image):
```

Comment:
Why not a simple str too and then we do the conversion?

Reply:
will add in a separate PR.